portfolio/src/main/webapp/photography.html

```diff
@@ -4,22 +4,39 @@
     <meta charset="UTF-8">
     <title>Photography</title>
     <link rel="stylesheet" href="style.css">
+        <style>
+        h1 {text-align: center;}
+p {text-align: center;}
+div {text-align: center;} 
```

Comment:
move those styles to the stylesheet

Reply:
I think the default is inherit, btw, so you could set this at a higher level (child elements will inherit their parent's text-align)

Reply:
Done